docs/getting-started/Legacy_ESLint_Setup.mdx

```diff
@@ -84,10 +84,40 @@ ESLint will lint all TypeScript compatible files within the current folder, and
 
 ## Next Steps
 
-We provide a plethora of powerful rules that utilize the power of TypeScript's type information. [Visit _Typed Rules_ for a setup guide](./Typed_Linting.mdx).
-
 If you're having problems getting this working, please have a look at our [Troubleshooting & FAQs](../troubleshooting/FAQ.mdx).
 
+### Additional Configs
+
+We provide two shared configs that we recommend you consider enabling:
+
+- [`strict`](../users/Shared_Configurations.mdx#strict): a superset of `recommended` that includes more opinionated rules which may also catch bugs.
+- [`stylistic`](../users/Shared_Configurations.mdx#stylistic): additional rules that enforce consistent styling without significantly catching bugs or changing logic.
+
+```js title=".eslintrc.cjs"
+/* eslint-env node */
+module.exports = {
+  extends: [
+    'eslint:recommended',
+    // Remove this line
```

Comment:
Question -should these be rendered as a diff rather than using comments to indicate removal and addition?

Reply:
JK - looking at the preview, there must be some automagical process that does exactly that. Neat!

Reply:
these comments are rendered by docusaurus as a diff.
if we used the `diff` language then you lose syntax highlighting.
we actually got them to add this feature so that we could have syntax highlighting AND diff highlighting.

![image of the thing](https://github.com/typescript-eslint/typescript-eslint/assets/7462525/884a14f7-8763-491b-a6fc-64eb7aac3ba4)

Reply:
That's really cool! That is exactly one of my gripes with writing diffs in markdown is that you lose syntax highlighting. Very nice 👏 👏